ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -142,7 +142,7 @@ export default class AddContact extends PureComponent {
         </div>
         <PageContainerFooter
           cancelText={this.context.t('cancel')}
-          disabled={Boolean(this.state.error)}
+          disabled={Boolean(this.state.error || !this.state.ethAddress)}
```

Comment:
I think we may want to move this `ethAddress` check to the `validate` method so all validation/errors are set in the same place.

Reply:
Hi @darkwing, I've tried that solution, but it didn't work well. The function `validate` checks only when we enter a value. In the task the button should be disabled when the field is empty. You can see the following video (when it is added in `validate` function)

https://user-images.githubusercontent.com/63151811/198064966-a3a6c1b0-f40d-41f4-ab59-85af61a140e8.mov